src/core-runtime.js

```diff
@@ -30,6 +30,14 @@ Globalize.locale = function( locale ) {
 	return this._locale;
 };
 
+Globalize._loadRuntimeFn = function( key ) {
+	var fn = Globalize[ key ];
+	if ( typeof fn === "function" ) {
+		fn.runtimeKey = key;
+	}
+	return fn;
+};
```

Comment:
Yeah, but how can you force a project using globalize and globalize-compiler to update the compiler when globalize is updated? If only globalize is updated, it will generate a different runtime key during compilation, and the runtime bundle won't find the correct function.